packages/ToggleSwitch/styles.jsx

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components'
+import { colorAccessibleGreen, colorGainsboro, colorShuttleGrey } from '@tds/core-colours'
+
+export const HiddenButton = styled.button({
```

Comment:
I'm wondering why there is a need to hide a button? Can a button be styled directly instead of creating a span?